src/components/Dashboard/TodoList.vue

```diff
@@ -330,7 +358,7 @@
             </v-card-text>
           </v-card>
 
-          <v-card v-else-if="isStatusError(task)" data-test-class="payment-unsuccessful">
+          <v-card v-else-if="isBcolError(task)" data-test-class="payment-unsuccessful">
```

Comment:
This is a duplicate "if" case of line 319. I think this line needs to be changed back to check for isStatusError(), to handle non-BCOL errors.

Reply:
eek, good catch

Reply:
Fixed.